Build.ps1

```diff
@@ -0,0 +1,6 @@
+#Prerequisite to be ran in a shell with VS2017 dev tools environment loaded
```

Comment:
If I got it right from your previous comment, this would be more a guidance than an actually build script to use?
In which case wouldn't it fit better in a README?